### PR TITLE
Add chartYAxes in scope directive

### PR DESCRIPTION
**Pull request by pane-e-design**
Add chartYAxes in scope directive for set chart axes ids when there are multiple y axes.

### CODE REVIEW

File: angular-chart.js [hunk 1]
```diff
@@ -261,14 +262,18 @@
...
       return {
         labels: labels,
         datasets: data.map(function (item, i) {
-          return angular.extend({}, colors[i], {
+          var dataset = angular.extend({}, colors[i], {
             label: series[i],
             data: item
           });
+          if (yaxis) {
+            dataset.yAxisID = 'y-axis-' + (i+1);
```
Comment (jtblin) [inline angular-chart.js]: Nit: use space between `i + 1`.

**Comment by jtblin** [comment]
LGTM apart from a minor nitpick but I'm not too familiar with this feature yet. Given it's a new feature, would it be possible that you add an example in `examples/chart.template.html`?

File: examples/app.js [hunk 2]
```diff
@@ -37,14 +37,36 @@
...
+            id: "y-axis-2",
+            type: "linear",
+            display: true,
+            position: "right"
+          }
+        ]
+      }
+    };
+
+    // $timeout(function () {
```
Comment (jtblin) [inline examples/app.js]: Please delete the comments in this case.

**Comment by jtblin** [comment]
The PR needs rebasing:

    git fetch upstream
    git rebase upstream/chartjs-2.0
    # resolve conflicts

Can you also delete the commented code and squash your commits into one?

File: examples/app.js [hunk 3]
```diff
@@ -37,14 +37,36 @@
...
+    //   $scope.labels = ['Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday', 'Sunday'];
```
Comment (jtblin) [inline examples/app.js]: Nit: it'd be great to keep these labels instead of the months ones.

File: examples/app.js [hunk 4]
```diff
@@ -37,14 +37,36 @@
       }
     };
 
-    $timeout(function () {
```
Comment (jtblin) [inline examples/app.js]: Should remove `$timeout` from the injected dependencies above: https://github.com/jtblin/angular-chart.js/pull/327/files#diff-416740f4b69f48495e4894828b2daea6L22

**Comment by pane-e-design** [comment]
@jtblin I squash commit into last one

**Comment by jtblin** [comment]
@pane-e-design I can see 6 commits now so they haven't be squashed, you added 3 more ;) 
See http://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git, I usually just do `git reset --soft HEAD~6 && git commit -m "Message"`.
Also the PR still need rebasing as it cannot be merged due to conflicts, not sure what the issue is.

**Comment by pane-e-design** [comment]
I had a difficulty to marge all commit into one...here i see now 8 commit but in last there are all modified files, if you can't merge this commit i submit another pull request rewrite all my change. I'm sorry for these problems.

**Comment by jtblin** [comment]
Yeah I can't merge it automatically as there are still conflicts. If I do it manually, then the commit will be in my name. I don't mind personally but that'd be a shame not to give you credit for that ;)

First, you need to add this repo as an upstream for your fork, see https://help.github.com/articles/configuring-a-remote-for-a-fork/.

Then you need to fetch upstream changes, and rebase:

```
git fetch upstream
git rebase upstream/chartjs-2.0
```

After that you need to manually solve the conflict. 

Then you need to squash your commits into one and force push to your repo:

```
git reset --soft HEAD~8 # I guess it's 8 now, make sure it's correct ;)
git commit -m "Add chartYAxes in scope directive for set chart axes ids when there are multiple y axes"
git push -f
```

You can try and submit another PR but you'll still need to rebase from upstream and squash your commits ;)

**Comment by pane-e-design** [comment]
I open new pull request #342 fetching upstream, rebase and with only one commit.

**Pull request closed without merging**